phase1-coordinator/src/coordinator.rs

```diff
@@ -613,10 +614,15 @@ impl Coordinator {
     /// Adds the given participant to the queue if they are permitted to participate.
     ///
     #[inline]
-    pub fn add_to_queue(&mut self, participant: Participant, reliability_score: u8) -> Result<(), CoordinatorError> {
+    pub fn add_to_queue(
+        &mut self,
+        participant: Participant,
+        participant_ip: Option<IpAddr>,
```

Comment:
Verifiers were the reason I made it optional, we don't necessarily need to track IPs for them afaik?

Reply:
If it makes the interface simpler, it doesn't necessarily hurt to track them?

Reply:
Unless verifiers need to be ordered by their IPs, there's no reason to store it. The optional value allows us to bypass that functionality when it isn't necessary. Do you have an explicit usecase in mind for tracking that value for verifiers as well?

Reply:
Nothing in particular, I guess it's a personal preference thing, I'd take storing an extra small value like an IP address over adding extra complexity in the code requiring us to handle and document the behaviour of  both the Some and the None through multiple layers of the code.  And as a bonus it also means it could potentially be used later. But I don't feel strongly about it, feel to resolve this conversation if you'd like :slightly_smiling_face: